common/src/update.rs

```diff
@@ -328,6 +328,19 @@ pub fn hex_schema<const N: usize>(gen: &mut SchemaGenerator) -> Schema {
     schema.into()
```

Comment:
github doesn't let me comment on the code above, but this is a bit of a surprising construction. In particular the `format` field in JSON Schema / OpenAPI isn't generally used for free-form text guidance--it seems to be for specific, well-known formats such as `email`, `uri`, or `uuid`.

I would have expected this to be something more like:

```json
{
  "type": "string",
  "pattern": "^[0-9a-fA-F]{64}",
}
```